boards/nucleo-l4r5zi/Kconfig

```diff
@@ -23,6 +23,8 @@ config BOARD_NUCLEO_L4R5ZI
     select HAS_PERIPH_TIMER
     select HAS_PERIPH_UART
 
+    select MODULE_PERIPH_LPUART if MODULE_STDIO_UART && HAS_PERIPH_LPUART
```

Comment:
Is the && HAS_PERIPH_LPUART needed here? Its already selected above?

Reply:
I suspect this will also lead to uart and lpuar being included.

Reply:
The && doesn't need to be added as the board config is where the `HAS_PERIPH_LPUART` is. 
I wanted to add that to make it a bit more verbose as it is a requirement, though the requirement is both given and consumed within the one symbol.

`PERIPH_UART` must be selected anyway for the `MODULE_PERIPH_LPUART` to even be exposed...

Would that be an additional requirement, or, since it is 'if'ed out it would throw an error if the `MODULE_PERIPH_LPUART` was trying to be selected but there was no `MODULE_PERIPH_UART`?

and I guess this is where things get confusing. 

MODULE_PERIPH_LPUART depends on MODULE_PERIPH_UART
MODULE_STDIO_UART selects MODULE_PERIPH_UART
therefore if MODULE_STDIO_UART selects MODULE_PERIPH_LPUART, we can assume MODULE_PERIPH_UART is selected.

However, if, for some reason, MODULE_STDIO_UART no longer selects MODULE_PERIPH_UART that would break that dependency.

I don't think there is a super clean solution.

I can remove the `HAS_PERIPH_LPUART` as it is declared above and this should only exist in boards that PERIPH_LPUART anyways and we can trust MODULE_STDIO_UART to being in all required modules and dependencies?